builtin/providers/digitalocean/resource_digitalocean_droplet_test.go

```diff
@@ -153,50 +159,46 @@ func testAccCheckDigitalOceanDropletRenamedAndResized(droplet *digitalocean.Drop
 	}
 }
 
-func testAccCheckDigitalOceanDropletAttributes_PrivateNetworkingIpv6(droplet *digitalocean.Droplet) resource.TestCheckFunc {
+func testAccCheckDigitalOceanDropletAttributes_PrivateNetworkingIpv6(droplet *godo.Droplet) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 
-		if droplet.ImageSlug() != "centos-5-8-x32" {
-			return fmt.Errorf("Bad image_slug: %s", droplet.ImageSlug())
+		if droplet.Image.Slug != "centos-5-8-x32" {
+			return fmt.Errorf("Bad image_slug: %s", droplet.Image.Slug)
 		}
 
-		if droplet.SizeSlug != "1gb" {
-			return fmt.Errorf("Bad size_slug: %s", droplet.SizeSlug)
+		if droplet.Size.Slug != "1gb" {
+			return fmt.Errorf("Bad size_slug: %s", droplet.Size.Slug)
 		}
 
-		if droplet.RegionSlug() != "sgp1" {
-			return fmt.Errorf("Bad region_slug: %s", droplet.RegionSlug())
+		if droplet.Region.Slug != "sgp1" {
+			return fmt.Errorf("Bad region_slug: %s", droplet.Region.Slug)
 		}
 
 		if droplet.Name != "baz" {
 			return fmt.Errorf("Bad name: %s", droplet.Name)
 		}
 
-		if droplet.IPV4Address("private") == "" {
-			return fmt.Errorf("No ipv4 private: %s", droplet.IPV4Address("private"))
+		if findIPv4AddrByType(droplet, "private") == "" {
+			return fmt.Errorf("No ipv4 private: %s", findIPv4AddrByType(droplet, "private"))
 		}
 
 		// if droplet.IPV6Address("private") == "" {
 		// 	return fmt.Errorf("No ipv6 private: %s", droplet.IPV6Address("private"))
 		// }
 
-		if droplet.NetworkingType() != "private" {
```

Comment:
i've removed this test because it does the same thing as the one above it (line 181):

```go
if findIPv4AddrByType(droplet, "private") == "" {
    return fmt.Errorf("No ipv4 private: %s", findIPv4AddrByType(droplet, "private"))
}
```